drivers/net/wireless/rtl8723bs/hal/hal_com_phycfg.c

```diff
@@ -2601,11 +2601,11 @@ phy_ConfigMACWithParaFile(
 
 	if ((pHalData->mac_reg_len == 0) && (pHalData->mac_reg == NULL))
 	{
-		rtw_merge_string(file_path, PATH_LENGTH_MAX, rtw_phy_file_path, pFileName);
+		rtw_merge_string(para_file_path, PATH_LENGTH_MAX, rtw_phy_file_path, pFileName);
```

Comment:
@dbnicholson suggested I change the variable name in this driver rather than rename the file_path() function. Not sure if this is the best idea.